### ISSUE TITLE
Focus state for Buttons

### ISSUE THREAD

**Issue by muratcorlu** [Trendyol/baklava#340]
Our buttons currently don't have focus state. This causes problems when we want to make our elements with keyboard navigation.

Baklava Buttons should be able to focus by keyboard and should have a visual indication that is focused now.

**Comment by buseselvi** [comment]
We added focused state for buttons to [design file](https://www.figma.com/file/RrcLH0mWpIUy4vwuTlDeKN/Baklava-Design-Guide?node-id=63%3A2852&t=Gs5dvI6GV9l8gyYT-1) 🚀
